ADOT EKS Addon

jasaws1048 opened the pull request:
*Issue #, if available:* ADOT add on feature in the EKS Cluster 

*Description of changes:*
The PR is to add the ADOT addon in the EKS cluster. CloudWatch Metrics is an AWS monitoring solution, and OpenTelemetry is a good compliment to it, so we have both as holistic solution. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

srinivasreddych commented:
@jasaws1048 You are missing entries into manifest files. Add entries into `example-dev` and others on how to enable this plugin?

dgraeber reviewed (commented):
Does the README need to be updated to reflect this change / support?

srinivasreddych commented:
Also you are missing to add the changelog?

jasaws1048 commented:
> @jasaws1048 You are missing entries into manifest files. Add entries into `example-dev` and others on how to enable this plugin?

Updated the example-dev

> Does the README need to be updated to reflect this change / support?

Updated the Readme.

> Also you are missing to add the changelog?

Added the changelog
